src/constants.h

```diff
@@ -60,4 +60,7 @@
 #define FRUIT_APPEAR_DURATION 2000
 #define FRUIT_SCORE_DISPLAY_DURATION 150
 
+#define NEXT_ROUND_ANIMATION_DURATION 500
+#define MAZE_BLINK_WHITE_BLUE_RATE 50 < 25
```

Comment:
What does this mean ?

Reply:
each 25 frames we show either the white or blue maze